openpype/hosts/max/plugins/publish/extract_thumbnail.py

```diff
@@ -0,0 +1,122 @@
+import os
+import tempfile
+import pyblish.api
+from openpype.pipeline import publish
+from pymxs import runtime as rt
+
+
+class ExtractThumbnail(publish.Extractor):
+    """
+    Extract Thumbnail for Review
+    """
+
+    order = pyblish.api.ExtractorOrder
+    label = "Extract Thumbnail"
+    hosts = ["max"]
+    families = ["review"]
+
+    def process(self, instance):
+        self.log.info("Extracting Thumbnail ...")
```

Comment:
I'd recommend removing this. We don't care whether a plug-in starts to process or not. If we're having issues detecting whether a plugin triggered or not I'd say that's a UX issue of the system itself, not the plugin.
```suggestion
```

If there's still a reason to keep it, then at least make this a debug log.